src/games/kalambury/board/GameBoard.js

```diff
@@ -71,6 +72,11 @@ const GameBoard = ({
     if (isDrawing) moves.UpdateDrawing(lines);
   }, [isDrawing, moves, lines]);
 
+  const handleChangePhrase = () => {
+    ChangePhrase();
+    setLines([]);
+  };
+
```

Comment:
And this is unrelated change done in #54

Reply:
yep 🙈